crates/matrix-sdk-crypto/src/olm/account.rs

```diff
@@ -604,28 +603,24 @@ impl Account {
     ///
     /// [1]: https://signal.org/docs/specifications/x3dh/#publishing-keys
     fn fallback_key_expired(&self) -> bool {
-        const FALLBACK_KEY_MAX_AGE: Duration = Duration::from_secs(3600 * 24 * 7);
+        // 604800000 = 3600 * 24 * 7 * 1000 (one week in milliseconds)
+        let fallback_key_max_age: UInt = uint!(604800000);
```

Comment:
Rust doesn't seem to want to let this be a `const`, and the macro doesn't allow calculations inside it.